packages/graphiql/package.json

```diff
@@ -16,6 +16,7 @@
   "module": "esm/index.js",
   "files": [
     "dist",
+    "src/locales",
```

Comment:
hmm, is there a reason these aren't being built I wonder? maybe we should configure webpack to copy them so that we can ensure relative resolutions work. overall I'm averse to the idea of publishing src to npm unless its an un-compiled package